src/api/verify.ts

```diff
@@ -6,7 +6,14 @@ import Logger from '../logger';
 import Session from '../session';
 import Storage from '../storage';
 
-import type { RadarTrackParams, RadarTrackVerifiedResponse } from '../types';
+import type { RadarStartTrackingVerifiedParams, RadarTrackParams, RadarTrackVerifiedResponse } from '../types';
+
+let tokenTimeoutId: any | null = null;
```

Comment:
Should we namespace these as `verified` instead of `token`?

Reply:
Also, it's probably preferable to mark things as optional instead of `null`
```ts
let tokenTimeoutId?: any;
```
Otherwise it needs to be explicitly set to `null`